data/formats-data.js

```diff
@@ -6108,7 +6108,7 @@ exports.BattleFormatsData = {
 		tier: "New",
 	},
 	tapulele: {
-		randomBattleMoves: ["moonblast", "psyshock", "calmmind", "focusblast", "aromatherapy"],
+		randomBattleMoves: ["moonblast", "psyshock", "calmmind", "focusblast", "hiddenpowerfire", "psychic", "thunderbolt"],
```

Comment:
Aromatherapy certainly doesn't give it any favors since it wants to attack a lot. Psychic is very good for damage under Psychic Terrain too. The rest can drop.